### PR TITLE
Gt/api20191101

### PR DESCRIPTION
**Pull request by gtracer**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

Modeled after this PR: https://github.com/Azure/azure-cli-extensions/pull/1010/commits/357d41a5f3e5e43ad8eb551f6c92ecda9d5ec0da

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ class ContainerServiceClient(MultiApiClientMixin, SDKClient):
             'open_shift_managed_clusters': '2018-09-30-preview',
             'container_services': '2017-07-01',
             'managed_clusters': '2019-10-01',
```
Comment (weinong) [inline src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py]: dont you need to remove this line?

File: src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py [hunk 2]
```diff
@@ -172,12 +180,9 @@ def container_services(self):
             from .v2017_07_01.operations import ContainerServicesOperations as OperationClass
         elif api_version == "2019-04-01":
             from .v2019_04_01.operations import ContainerServicesOperations as OperationClass
-        elif api_version == '2019-08-01':
-            from .v2019_04_01.operations import ContainerServicesOperations as OperationClass
-            api_version = "2019-04-01" #2019-08-01 generated Python SDK contains no list_orchestrator op
-        elif api_version == '2019-10-01':
+        elif api_version in ['2019-08-01', '2019-10-01', '2019-11-01']:
```
Comment (weinong) [inline src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py]: nice trick!

File: src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py (hunk 1 above)
Comment (gtracer) [reply src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/container_service_client.py]: Yes, I think you are right - will update

**Comment by gtracer** [comment]
Looks like 2020_01 in now in extensions - abandoning

**Pull request closed without merging**